src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Api/ISegment.cs

```diff
@@ -49,6 +49,11 @@ public interface ISegment : ISpan
         /// </summary>
         void End();
 
+        /// <summary>
+        /// Ends this StackExchange.Redis transaction segment.
+        /// </summary>
+        void EndStackExchangeRedis();
```

Comment:
Should this also be in an experimental interface? Many of the related API methods are in an experimental namespace.

Reply:
@jaffinito - thoughts? I'm not familiar with what we've flagged as Experimental in the past.